### ISSUE TITLE
Removing vdb_chunks.json whens starting a new run in the same WORKING_DIR

### ISSUE THREAD

**Issue by yagneshgooglegithub** [gusye1234/nano-graphrag#103]
I think `WORKING_DIR/vdb_chunks.json` should also be removed while starting a new run in the same directory. I had issues when querying because a specific chunk ID was not present in kv_chunks_text_units.json, but was there in vdb_chunks.json. Maybe new additions are appended, and old ones are getting referred?

**Comment by Vaccy-Zhu** [comment]
Yes, it's pretty necessary. If you don't delete vectors cache first, old eneities will be referred.

**Comment by yagneshgooglegithub** [comment]
> Yes, it's pretty necessary. If you don't delete vectors cache first, old eneities will be referred.

Thanks for the reply.

**Comment by Buzeg** [comment]
Same here, any solution?
